### ISSUE TITLE
RBR cannot communicate with SpaceMonkey on Henk's PC.

### ISSUE THREAD

**Issue by PHARTGAMES** [PHARTGAMES/SpaceMonkey#3]
I think this may be an issue with the same port used by multiple apps. Need to take some time with Henk to work it out.

**Comment by simracingstudio** [comment]
You need a udp port forwarded
Some games locks to only 1 interface to listen the udp packets

**Comment by PHARTGAMES** [comment]
> You need a udp port forwarded
> Some games locks to only 1 interface to listen the udp packets

Yeah I think that's his issue, but he has been running wireshark to check, and port forwarding with simhub...

I think his system is complicated by multiple softwares and needs to try just use one thing at a time to figure out what's eating the port.
